pcsx2/USB/usb-msd/usb-msd.cpp

```diff
@@ -75,7 +75,6 @@ namespace usb_msd
 			struct usb_msd_csw csw;
 			enum USBMSDMode mode;
 			uint32_t data_len;
-			uint32_t residue;
```

Comment:
if you're removing this you might wanna bump the savestate version.

Reply:
Bummer, worth doing the change since it will need a save state bump?

Reply:
eh, probably not.